### PR TITLE
Removing older Confidential Ledger package name

### PR DESCRIPTION
**Pull request by christothes**

### CODE REVIEW

File: _data/releases/latest/dotnet-packages.csv [hunk 1]
```diff
@@ -18,7 +18,6 @@
 "Azure.Communication.PhoneNumbers","1.0.1","","Communication Phone Numbers","Communication","communication","","","client","true","","04/26/2021","","","",""
 "Azure.Communication.Sms","1.0.1","","Communication SMS","Communication","communication","","","client","true","","03/29/2021","active","","",""
 "Azure.Security.ConfidentialLedger","","1.0.0-beta.2","ConfidentialLedger","ConfidentialLedger","confidentialledger","","","client","true","","","","","",""
-"Azure.Storage.ConfidentialLedger","","1.0.0-beta.1","ConfidentialLedger","ConfidentialLedger","confidentialledger","","","client","true","","","","","",""
```
Comment (jongio) [inline _data/releases/latest/dotnet-packages.csv:21]: Set Hide to true instead of removing.  The automation will add it back.

**Review by jongio — CHANGES_REQUESTED** [review]

File: _data/releases/latest/dotnet-packages.csv (hunk 1 above)
Comment (weshaggard) [reply _data/releases/latest/dotnet-packages.csv:21]: Look at the first line but it is the 3 field from the end.

Comment (christothes) [reply _data/releases/latest/dotnet-packages.csv:21]: We should - I can work with Alex on that.

Comment (weshaggard) [reply _data/releases/latest/dotnet-packages.csv:21]: I've just unlisted the package on nuget so we can just remove this entry now without it coming back.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**